nubilanding/src/components/blog/form.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+class Form extends React.Component {
+  state = { email: '' };
+  constructor(args) {
+    super(args)
+    this.state = {
+
+    }
+  }
+
+  onChange(e) {
+    this.setState({
+      [e.target.email]: e.target.value
+    })
+  }
+  render() {
```

Comment:
Presta atención a los espaciados entre los métodos para facilitar la legibilidad.
```suggestion
  }

  render() {
```